src/views/CreateNewListing.vue

```diff
@@ -44,8 +36,7 @@ import FileMetadata from '@/components/listing/FileMetadata.vue'
 
 import Dropzone from 'dropzone'
 
-import '@/assets/style/views/list.sass'
-import '@/assets/style/components/file-uploader.sass'
+import '@/assets/style/views/create-new-listing.sass'
```

Comment:
I agree with the convention. Possibly a good amount of these changes moving forward